models/OrderHeader.js

```diff
@@ -0,0 +1,28 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('OrderHeader', {
+		order_user_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			references: {
+				model: 'User',
+				key: 'id'
+			}
+		},
+		order_supplier_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false
+		},
+		order_item_count: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false
+		},
+		order_status: {
```

Comment:
Should there be an order status table? What are the different statuses? Where to keep track of that list so that everybody is on the same page?

Reply:
We can choose to add a code table (like the UserRole), but just adds to the complexity.
Let us talk about it on Saturday.

For now not making any change